go.mod

```diff
@@ -69,6 +69,7 @@ require (
 	k8s.io/kube-proxy v0.0.0-00010101000000-000000000000 // indirect
 	k8s.io/kubectl v0.0.0-20191003004222-1f3c0cd90ca9
 	sigs.k8s.io/controller-runtime v0.2.2
+	sigs.k8s.io/kind v0.5.1 // indirect
```

Comment:
this should not be here and pulls in dependencies that may not be ok for istio/istio

Reply:
Ya not sure why this is present will fix this after dinner tonight.

Reply:
Ya not sure why this is present will fix this after dinner tonight.